src/modules/job-list/job_data.c

```diff
@@ -266,13 +266,12 @@ static int parse_jobspec (struct job *job, const char *s, bool allow_nonfatal)
     json_error_t error;
```

Comment:
In the commit message for "job-list: do not return error when nonfatal set" there's a duplicate jobspec word: "illegal json jobspec jobspec"

Reply:
oops! thanks.  i'll fix up then set MWP.